spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitTemplateConfigurer.java

```diff
@@ -64,15 +64,15 @@ public RabbitTemplateConfigurer(RabbitProperties rabbitProperties) {
 	 * converter should be used.
 	 * @param messageConverter the {@link MessageConverter}
 	 */
-	protected void setMessageConverter(MessageConverter messageConverter) {
+	public void setMessageConverter(MessageConverter messageConverter) {
 		this.messageConverter = messageConverter;
 	}
 
 	/**
 	 * Set the {@link RabbitRetryTemplateCustomizer} instances to use.
 	 * @param retryTemplateCustomizers the retry template customizers
 	 */
-	protected void setRetryTemplateCustomizers(List<RabbitRetryTemplateCustomizer> retryTemplateCustomizers) {
+	public void setRetryTemplateCustomizers(List<RabbitRetryTemplateCustomizer> retryTemplateCustomizers) {
```

Comment:
This is needed [here](https://github.com/spring-cloud/spring-cloud-stream-binder-rabbit/pull/323/files#diff-91292975db6a0350e86f703102cb7596bc9f5741b8372ef5fcadf6b156c09ce0R103).

Reply:
Can you please add `@since 2.6.0` to the javadoc?